cla_backend/libs/eligibility_calculator/cfe_civil/age.py

```diff
@@ -0,0 +1,12 @@
+import datetime
+
+
+def translate_age(today, facts):
+    if hasattr(facts, "is_you_or_your_partner_over_60") and facts.is_you_or_your_partner_over_60:
+        age = 70
```

Comment:
@starswan What do you think of adding a reasoning for choosing 70? Also should we use constant for age ?